app/controllers/transcriptions_controller.rb

```diff
@@ -41,6 +50,17 @@ def update
     render jsonapi: @transcription
   end
 
+  def unlock
+    @transcription = Transcription.find(params[:id])
+    authorize @transcription, :update?
+    if @transcription.locked_by != current_user.login
+      raise ActionForbiddenError,
+            "You are not allowed to unlock this transcription. Transcription is locked by #{@transcription.locked_by} and can only be unlocked by this user."
+    end
+
+    @transcription.update!(locked_by: nil, lock_timeout: nil)
```

Comment:
This should be an unlock method on the model, move the functionality there so the controller reads nicely and the model knows how to unlock itself.
```suggestion
    @transcription.unlock!
```

Reply:
Makes sense 👍